loc/migrations/0020_auto_20200401_1345.py

```diff
@@ -43,6 +43,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='landlorddetails',
             name='address',
-            field=models.CharField(help_text='The full mailing address for the landlord. This is a LEGACY field that we prefer not to use if possible, e.g. if the more granular primary/secondary line and city/state/zip details are available on this model.', max_length=1000),
+            field=models.CharField(verbose_name="LEGACY address", help_text='The full mailing address for the landlord. This is a LEGACY field that we prefer not to use if possible, e.g. if the more granular primary/secondary line and city/state/zip details are available on this model.', max_length=1000),
```

Comment:
`verbose_name` doesn't affect the actual DB schema so I've just hacked the most recent migration to include it, which obviates creating another migration.